Northstar.Client/mod/scripts/vscripts/ui/menu_ns_modmenu.nut

```diff
@@ -457,23 +457,18 @@ vector function GetControlColorForMod( string modName )
 		switch ( GetConVarInt( "colorblind_mode" ) )
 		{
 			case 1:
-				return <0.973, 0.863, 0>
 			case 2:
-				return <1,0.835,0.6>
 			case 3:
-				return <0.459,0.923,1>
 			default:
 				return <0,1,0>
 		}
 	else
 		switch ( GetConVarInt( "colorblind_mode" ) )
 		{
 			case 1:
-				return <0.561, 0.494, 0.118>
 			case 2:
-				return <0.631,0.471,0>
+				return <0.29,0,0.57>
```

Comment:
I'm confused how this helps in case of colour blindness? Wouldn't it be easier to add a text or icon if a mod is disabled? ^^

At least in testing it still looks like just switching between green and red.

Reply:
Have you tried turning on red / green color blind mode in the settings?

Reply:
Oooh nevermind I see it now, nice :D

Reply:
I'm still in favour of using icons or sth additionally but looks good enough ig ^^

Reply:
What icon would you suggest to communicate that the mod is disabled?
I'd like to avoid cluttering the screen with `(DISABLED)` because that makes the list hard to read

Reply:
I suppose a checkmark and cross? As I said though, not really important to get the PR merged compared to the stuff printed to logs mentioned in the comment below.

Reply:
I'm going to remove the prints tomorrow.
I don't think there are rui assets with a cross? Bounty hunt had a checkmark iirc